Empire/SDK/EmpireSdk.php

```diff
@@ -211,7 +212,65 @@ public function queryContentIdMap( $first, $skip ) {
             )
         );
         $result = $this->runQuery( $gql );
-        return $result->data->contentIdMap;
+        return $result['data']['contentIdMap'];
+    }
+
+    public function queryAdConfig() {
+        $gql = ( new Query( 'appAds' ) );
```

Comment:
One thing I hadn't considered—the WP plugin lives in its own repo and we always make the GQP API backwards-compatible, but we aren't really testing the plugin when we go to deploy API changes.

We may want to consider putting some safety checks into the build pipeline to test the plugin as well when we make changes to the API. This already implicitly happens on the frontend since the API/frontend are in a monorepo where all our tests run.

What would be _really_ cool is to have a test WP site that pulls the latest plugin and runs tests to make sure ads display etc. as part of the API test pipeline.

Reply:
Yeah, great idea! Created task for this https://empireio.atlassian.net/browse/EP-708